### PR TITLE
Repaths all /obj/item/weapon/melee/ to /obj/item/weapon/

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

There was no /obj/item/weapon/melee item, ctrl-clicking would just redirect to chloroform, this changes nothing

Wow writing this PR is lagging out my PC
# Explain why it's good for the game

It will do nothing but cause merge conflicts
The /melee segment doesn't change anything about the code, it is just there

# Changelog

:cl:
code: Repaths melee weapons
qol: You can see the melee and throw force of more items
/:cl:

### CODE REVIEW

**Review by harryob — APPROVED** [review]
yes boss conflict those prs

**Comment by Ultimate-Fluff** [comment]
I thought _my_ PRs modified a lot of files...

**Comment by realforest2001** [comment]
image
this is what melee is for

File: code/game/objects/items/weapons/weapon.dm [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
-			strong_text = "strong"
+			strong_text = "a strong"
 		if(MELEE_FORCE_STRONG to MELEE_FORCE_VERY_STRONG)
-			strong_text = "very strong"
+			strong_text = "a very strong"
 		if(MELEE_FORCE_VERY_STRONG to INFINITY)
-			strong_text = "inhumanely strong"
-	. += SPAN_INFO("This weapon looks [strong_text].")
+			strong_text = "an inhumanely strong"
+	. += SPAN_INFO("[src] would be [strong_text] weapon if you were hit someone with it.")
```
Comment (harryob) [inline code/game/objects/items/weapons/weapon.dm]: make nothing show up if it's less than weak

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/weapon.dm (hunk 1 above)
Comment (harryob) [reply code/game/objects/items/weapons/weapon.dm]: ^ this guys an idiot btw

**Pull request merged**